src/lib/DecodedMessage.ts

```diff
@@ -1,8 +1,20 @@
-export type DecodedMessage = {
+export class DecodedMessage {
```

Comment:
Why did we need to make this a class?

Reply:
@nakajima Maybe this is just my lack of knowledge of typescript but I couldn't figure out how to make a DecodeMessage value on this line https://github.com/xmtp/xmtp-react-native/pull/60/files#diff-c5227210ea741b70a7f1c730d3dc81ff6c90156a21cee65f6fbb29f86099c9f8R67. I kept getting

```
'DecodedMessage' only refers to a type, but is being used as a value here.
```